src/editorApp/checkPage.js

```diff
@@ -5,11 +5,12 @@ import { showNotice } from './../common/helpers';
 
 const API_URL = edac_editor_app.edacApiUrl;
 
+
 let HEADERS;
-if (typeof (edacp_full_site_scan_app) === 'undefined') {
+if (typeof (edacpFullSiteScanApp) === 'undefined') {
```

Comment:
This misnamed variable is the root cause of the issue b/c the user/pass were not being passed so auth failed.